cfme/tests/containers/test_autotagging_providers.py

```diff
@@ -0,0 +1,198 @@
+from collections import namedtuple
+import fauxfactory
+import pytest
+from random import choice
+from traceback import format_exc
+
+from cfme.configure.configuration.region_settings import MapTags
+from cfme.containers.provider import ContainersProvider
+from cfme.containers.node import Node
+from cfme.containers.replicator import Replicator
+from cfme.containers.image import Image
+from cfme.containers.pod import Pod
+from cfme.containers.project import Project
+from cfme.containers.route import Route
+
+from utils import testgen
+from utils.version import current_version
+from utils.wait import wait_for
+from utils.log import logger
+
+
+pytestmark = [
+    pytest.mark.uncollectif(lambda: current_version() < "5.8"),
+    pytest.mark.usefixtures('setup_provider_modscope'),
```

Comment:
**OPTIONAL:** modscope for providers shouldn't be necessary as the check func is quick enough now that we don't need to worry.